class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace BracketSpace\Notification\Defaults\Trigger\WordPress;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+use BracketSpace\Notification\Abstracts;
+
+class ActivePlugin extends Abstracts\Trigger{
+
+	private $plugin;
+
+	public function __construct(){
+
+		parent::__construct( 'wordpress/active_plugin',  __( 'Plugin Active', 'notification' ) );
+
+		$this->add_action( 'activated_plugin', 10, 2 );
+		$this->set_group( __( 'WordPress', 'notification' ) );
+		$this->set_description( __( 'Fires when plugin can be active', 'notification' ) );
```

Comment:
The description doesn't feel right. Did you mean something like `Fires when a plugin is activated`